Lib/test/test_memoryview.py

```diff
@@ -71,6 +71,21 @@ def setitem(value):
         m = None
         self.assertEqual(sys.getrefcount(b), oldrefcount)
 
+    def test_setitem_readonly_and_writable_buffer(self):
+        if not self.rw_type:
+            self.skipTest("no writable type to test")
```

Comment:
Even if it's not writable, you should still be able to cast to `readonly=True`.

Reply:
Should I test both (writable and not writable) separately?